examples/crds/ciliumnodes.yaml

```diff
@@ -163,6 +163,10 @@ spec:
                     minimum: 0
                     type: integer
                 type: object
+              nodeidentity:
+                description: NodeIdentity is the Cilium numeric identity allocated for the node, if any.
+                format: int64
```

Comment:
Does CRD provide a way to restrict this further?

In general, we tend to use 24 bits for security identities with the lowest 16 bits representing the in-cluster identity and the 8 bits above representing the cluster. I'm guessing we will ignore the 40 bits above that, so I'm wondering about the consequences. Is it fine because in practice this is populated by the agent?

Reply:
Agent itself uses uint32 type, but I went with uint64 for the CRD as there are other types in the CRD using uint64 for the numeric identity (e.g., `AllowedIdentityTuple.Identity`). Given this precedent I'm inclined to keep this as-is.